src/style.css

```diff
@@ -1,3 +1,125 @@
-body {
+/* #region CSS Reset */
+html,
+body,
+div,
+span,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+a,
+em,
+img,
+footer,
+header,
+nav,
+ul,
+ol,
+li {
   margin: 0;
 }
+
+/* #endregion */
+
+/* #region Page  */
+.html {
+  font-family: Roboto, sans-serif;
+
+  --header-height: 60px;
+  --moyo-blue-color: #00acdc;
+}
+
+
+/* #endregion */
+
+/* #region Fonts  */
+.roboto-regular {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+.roboto-bold {
+  font-family: Roboto, sans-serif;
+  font-weight: 700;
+  font-style: normal;
+}
+
+.roboto-regular-italic {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: italic;
+}
+
+.roboto-bold-italic {
+  font-family: Roboto, sans-serif;
+  font-weight: 700;
+  font-style: italic;
+}
+
+/* #endregion */
+
+/* #region Header */
+.header {
+  height: var(--header-height);
```

Comment:
The height must be for links, not for header
```suggestion
```

Reply:
Didnt find that requirement in description, but ok